Client/Frontend/Browser/Search/SearchEngines.swift

```diff
@@ -250,7 +250,8 @@ class SearchEngines {
 
     /// Get all bundled (not custom) search engines, with the default search engine first,
     /// but the others in no particular order.
-    class func getUnorderedBundledEnginesFor(selected: [String] = []) -> [OpenSearchEngine] {
+    class func getUnorderedBundledEngines(selected: [String] = [],
```

Comment:
I think function can be renamed like
`class func getUnorderedBundledEngines(for selectedEngines: [String] = [],  isOnboarding: Bool) `